usecases/config/environment.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	DefaultRaftPort             = 8300
 	DefaultRaftInternalPort     = 8301
+	DefaultRaftGRPCMaxSize      = 1024 * 1024 * 1024
```

Comment:
This is the most important question. Should `DefaultRaftGRPCMaxSize` be set to something quite large like `1GB` so that things "just work" for users until they have very large messages being sent which exceed this or should we leave the default low and force users to pick a higher one to avoid performance surprises?

Reply:
i'd say let's over assign to `1G` to make it able to handle large msgs. it won't consume any memory or CPU anyway or at least until it's used fully the whole 1G. so i don't see any issue in over assign